src/core/execute.c

```diff
@@ -453,6 +453,12 @@ int exec_spawn(
          * handoff timestamp. */
         dual_timestamp_now(&start_timestamp);
 
+        r = 0;
```

Comment:
rather than setting r=0 beforehand, simply move the r check inside the next if block, following our usual style

Reply:
(same below)

Reply:
DONE